### PR TITLE
[DX-146] Remove free tier from the docs

### PR DESCRIPTION
**Pull request by AMM55**
Remove cloud free from the docs

### CODE REVIEW

**Review by marksou — APPROVED** [review]

File: tyk-docs/content/tyk-cloud/account-&-billing/plans.md [hunk 1]
```diff
@@ -21,14 +21,13 @@ Here's an overview of all of the available plans:
...
 |-------------------|-------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | 5-week free trial | This is for POC’s and those testing the Tyk platform.                               | The Tyk Cloud free is 5 weeks so that you would have enough time to work your way through all the features. With this, you get **5-week access** to the full Tyk Stack. For the 5-week trial plan, the quotas are the same as for our Enterprise Global plan. You can always request a longer trial period or talk to support if you need help. |
```
Comment (letzya) [inline tyk-docs/content/tyk-cloud/account-&-billing/plans.md]: Can you also change the text here "tyk cloud free is 5 weeks" - this is the wrong name. similar to my PR https://github.com/TykTechnologies/tyk-docs/pull/2056/files

**Review by michellefernando9 — APPROVED** [review]

File: tyk-docs/content/tyk-cloud/account-&-billing/plans.md [hunk 2]
```diff
@@ -20,15 +20,14 @@ Here's an overview of all of the available plans:
...
-| Cloud Free        | This is for Makers, POC’s & Hackers.                                                | This plan includes all the same features included in the other plans but with limitations. There is no choice of region, must be a **fixed Cloud Free region (EU West)**. In case you want an upgrade, data migration would be involved.                                                                                                          |
+| 5-week free trial | This is for POC’s and those testing the Tyk platform.                               | The Tyk Cloud has 5-week free trial so that you would have enough time to work your way through all the features. With this, you get **5-week access** to the full Tyk Stack. For the 5-week trial plan, the quotas are the same as for our Enterprise Global plan. You can always request a longer trial period or talk to support if you need help. |
```
Comment (letzya) [inline tyk-docs/content/tyk-cloud/account-&-billing/plans.md]: Do we need the “The” in the beginning?

**Review by letzya — CHANGES_REQUESTED** [review]
Agreed to remove "The"

**Review by letzya — APPROVED** [review]

**Pull request merged**